enhancements/operator-sdk/downstream-operator-sdk.md

```diff
@@ -0,0 +1,366 @@
+---
+title: downstream-operator-sdk
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@joelanford"
+  - "@asmacdo"
+  - "@fabianvf"
+  - "@camilamacedo86"
+  - "@shawn-hurley"
+  - "@gallettilance"
+  - "@kevinrizza"
+approvers:
+  - "@joelanford"
+  - "@asmacdo"
+  - "@fabianvf"
+creation-date: 2020-09-10
+last-updated: 2020-09-10
+status: implementable
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Downstream Operator SDK
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The Operator SDK currently ships the Ansible and Helm Operator base images
+downstream. The purpose of this enhancement is to add a supported Operator SDK
+base image as well as an operator-sdk binary.
+
+## Motivation
+
+Deliver a properly supported Operator SDK downstream to support OpenShift
+developers.
+
+### Goals
+
+To deliver a new base image with the Operator SDK that can be used by operator
+developers. Deliver an operator-sdk binary similar to how oc is delivered to
+customers. Continue to deliver the Ansible and Helm operator base image.
+
+Another goal is to add a script to make it easier for developers to bring in
+upstream releases into the downstream.
+
+
+### Non-Goals
+
+We will not deliver an RPM based operator-sdk package as part of this. SDK
+downstream scaffolding files to use downstream images this will be done when
+Phase 2 plugins are done. For now folks will have to modify their operators to
+pull from downstream images.
```

Comment:
Could we not just add a replace in the script to update the Dockerfiles? see that we just need to update the : 
- https://github.com/operator-framework/operator-sdk/blob/master/internal/plugins/ansible/v1/scaffolds/internal/templates/dockerfile.go#L50
AND
- https://github.com/operator-framework/operator-sdk/blob/master/internal/plugins/helm/v1/scaffolds/internal/templates/dockerfile.go#L43